Travel Scheduler/APIManager.m

```diff
@@ -173,8 +173,10 @@ -(void)getDistanceFromOrigin:(NSString *)origin toDestination:(NSString *)destin
         }
         else {
             NSDictionary *rowsDictionary = [jSONresult valueForKey:@"rows"];
-            NSDictionary *distanceDurationDictionary = rowsDictionary[@"elements"][0];
-            completion(distanceDurationDictionary, nil);
+            NSDictionary *distanceDurationDictionary = [rowsDictionary valueForKey:@"elements"];
+            //rowsDictionary[@"elements"];
+            NSDictionary *durationDictionary = [distanceDurationDictionary valueForKey:@"duration"];
```

Comment:
I am not exactly sure what you did here, but I think it is usually better to use the format dictionaryName[@"keyName"] to access a value from the dictionary, since we do not call a method

Reply:
Yeah, I agree. For some reason, it wasn't working though so I just did the quickest fix. Not going to override these changes over yours.